sd_card/storage.h

```diff
@@ -35,6 +35,14 @@ typedef struct {
 	const void * audio_buffer;
 } audio_source_info_t;
 
+typedef struct __attribute__((__packed__)) {
+	uint32_t seconds;
+	uint16_t milliseconds;
```

Comment:
Sorry to be so nitpicking with this PR, can you change the seconds and milliseconds to be `uint64_t timestamp;`  That is how the timestamp is saved for the IMU sensors (imu_sample_t) and I'd prefer it to be consistent. Or did I get that wrong?